appservice/src/deploy/runWithZipStream.ts

```diff
@@ -80,6 +80,11 @@ async function getFilesFromGlob(folderPath: string): Promise<string[]> {
     const ignorePattern: string | string[] = zipDeployConfig.get<string | string[]>('zipIgnorePattern') || '';
     const filesToIgnore: string[] = await globby(ignorePattern, globOptions);
 
+    ext.outputChannel.appendLog(localize('zipIgnoreFilesMsg', 'Ignoring files from .vscode/settings.json \"appService.zipIgnorePattern\"'), { resourceName: client.fullName });
```

Comment:
I wouldn't list the file name `.vscode/settings.json` since it could be set elsewhere, like in global user settings